Automattic/wp-calypso
# Translate Verbum Block Editor Package

## Proposed Changes

This downloads Gutenberg translations from .org's GlotPress and bundles it with VBE to be deployed in widgets.wp.com. This PR will not make the editor translate well. It just creates the translations file to be consumed later. 

## Testing Instructions
1. cd into `packages/verbum-block-editor`.
2. run `yarn build`.
3. run `yarn dev --sync`.
4. There should be many JSON files in `/dist/languages`.
